vmdb/app/models/miq_expression.rb

```diff
@@ -31,6 +33,7 @@ class MiqExpression
     OntapStorageVolume
     OntapVolumeMetricsRollup
     PolicyEvent
+    Provider
```

Comment:
We may want to change the dictionary entry for ExtManagementSystem (and EmsCloud, EmsInfra) to maybe Manager.  With the new Provider/Manager concept, technically they aren't the providers. cc @dclarizio